### ISSUE TITLE
[Feature request] GI-1.0

### ISSUE THREAD

**Issue by trsh** [turanszkij/WickedEngine#584]
Amd recently published a paper on a 'new' GI technique https://gpuopen.com/download/publications/GPUOpen2022_GI1_0.pdf . Its work on top of Daniel Wright's SSRC (https://advances.realtimerendering.com/s2021/index.html#_mrnver3hf0ag) what is the main component powering UE5 Lumen. The technique promises good balance between quality and performance that is scalable and good set of features. In the introduction they present the advantages over current techniques like DDGI, tracing with ReSTIR, etc. 

I see lot of GI methods already implemented here with high quality and professionalism. Maybe this could be another one?

**Comment by turanszkij** [comment]
Yes, implementing better GI is always on the todo list. This presentation will have plenty useful ideas.
